src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/TouchstoneRouteConfig.kt

```diff
@@ -13,12 +13,17 @@ object TouchstoneRouteConfig : RouteConfig
     private val permissions = setOf("*/touchstones.read")
     private val scenarioPermissions = permissions + setOf("*/scenarios.read", "*/coverage.read")
     private val demographicPermissions = permissions + setOf("*/demographics.read")
+    private val responsibilityPermissions = permissions + setOf("*/scenarios.read", "*/responsibilities.read")
```

Comment:
I don't think this matches the spec. Unless I'm reading the wrong bit, above you said "Required permissions: `touchstones.read`, `responsibilities.read`"

Reply:
Ah you're right. I think this stricter condition is correct, because the scenarios are returned with the responsibilities. Will update spec.